inventory.py

```diff
@@ -41,14 +41,42 @@
 '''
 import os
 import argparse
-import ConfigParser
 import re
 import sys
+from test.test_sax import start
 try:
     import json
 except ImportError:
     import simplejson as json
-
+try:
+    # For Python >= 3.x
+    import configparser
+except:
+    # For Python 2.x
+    import ConfigParser as configparser
+
+"""
+Modified ConfigParser that allows ':' in keys and only uses '=' as separator.
+We need the : to be able to specify groups of Ansible hosts like this: compute_nodes[01:16]
+"""
+class MyConfigParser(configparser.SafeConfigParser):
```

Comment:
Maybe it's nessecary but why are you subclassing configparser instead of sublassing whatever ansible uses to parse inventory files?